### ISSUE TITLE
Upload several files in one shot

### ISSUE THREAD

**Issue by mariePredicsis** [PredicSis/predicsis-python-sdk#94]
Now, you need to upload 1 dataset per 1 dataset using the method project.upload_files('name of frame','path to to the file',is_central=True train=True, test=False) . Same thing for test, and again once per peripheral table.
Is it possible to upload everything with one command line. Or the user could be able to define a python dictionary that could be passed (later) as the upload method parameter.

```
>> project_dataset = {
...  "train": "/path/to",
...  "test": "/path/to",
...  "framename": "master",
...  "peripherals": [
...    {
...      "train": "/path/to",
...      "test": "/path/to",
...      "tablename": "orders"
...    },
...   {
...      "train": "/path/to",
...      "test": "/path/to",
...      "tablename": "emails"
...    },
...   {
...      "train": "/path/to",
...      "test": "/path/to",
...      "tablename": "visitedpages"
...    }
...  ]
...}
```

First level of the dictionary would be the central table (compulsory), and then, the peripheral, a level under, would be facultative.

And then, it could be used directly to upload the data:
```
>> prj.upload_data(project_dataset)
```
@jeanbaptistepriez That does not forbidden, for later, to upload datasets independently of projects. PredicSis.upload_data(project_dataset).

**Comment by jeanbaptistepriez** [comment]
Do you agree with those methods:
![capture d ecran 2017-03-20 a 14 05 12](https://cloud.githubusercontent.com/assets/17195729/24100929/902790dc-0d76-11e7-9686-f6d8e0f4e4ee.png)
![capture d ecran 2017-03-20 a 14 05 21](https://cloud.githubusercontent.com/assets/17195729/24100933/92fc69b8-0d76-11e7-98dc-f5013739618c.png)
![capture d ecran 2017-03-20 a 14 05 28](https://cloud.githubusercontent.com/assets/17195729/24100937/96437e4a-0d76-11e7-870b-b07990ae6bed.png)

**Comment by jeanbaptistepriez** [comment]
This is implemented in #100
